K2Bridge/Visitors/Aggregations/MetricAggregationVisitor.cs

```diff
@@ -33,6 +34,29 @@ public void Visit(CardinalityAggregation cardinalityAggregation)
             cardinalityAggregation.KustoQL = $"{EncodeKustoField(cardinalityAggregation.Key)}={KustoQLOperators.DCount}({EncodeKustoField(cardinalityAggregation)})";
         }
 
+        /// <inheritdoc/>
+        public void Visit(ExtendedStatsAggregation extendedStatsAggregation)
+        {
+            Ensure.IsNotNull(extendedStatsAggregation, nameof(extendedStatsAggregation));
+            EnsureClause.StringIsNotNullOrEmpty(extendedStatsAggregation.Field, extendedStatsAggregation.Field, ExceptionMessage);
+
+            var sep = AggregationsConstants.MetadataSeparator;
+            var key = $"['{extendedStatsAggregation.Key}{sep}extended_stats']";
+            var average = $"{KustoQLOperators.Avg}({EncodeKustoField(extendedStatsAggregation)})";
+            var standard_deviation = $"{KustoQLOperators.StDev}({EncodeKustoField(extendedStatsAggregation)})";
```

Comment:
camel case

Reply:
Done

Reply:
Now it seems that all of them are snake_case instead of camel case:
var sum_of_squares = $"{KustoQLOperators.Sum}({KustoQLOperators.Pow}({EncodeKustoField(extendedStatsAggregation)}, 2))";
            var variance_population = $"{KustoQLOperators.VariancePopulation}({EncodeKustoField(extendedStatsAggregation)})";
            var variance_sampling = $"{KustoQLOperators.Variance}({EncodeKustoField(extendedStatsAggregation)})";
            var standard_deviation_population = $"{KustoQLOperators.StDevPopulation}({EncodeKustoField(extendedStatsAggregation)})";
            var standard_deviation_sampling = $"{KustoQLOperators.StDev}({EncodeKustoField(extendedStatsAggregation)})";

please fix

Reply:
Done